### DISCUSSION TITLE
Can Space-eye data work for Pakistan?

### DISCUSSION THREAD

**Discussion by mursaleen-bkk** [microsoft/farmvibes-ai#64] [Q&A] [answered]
Sir, I want to monitor the agriculture in Pakistan (specifically harvesting of the crops). But from December 2022 to February 2023 we have clouds and frost in our study area. For that I want to use the space-eye data. I downloaded space-eye data from space-eye download workflow. But What I Got that it give me space-eye raster for each day. And most of the rasters are same and have no difference from each other. 
So, I want to understand that how Space-eye Model works. And is it suitable to use for Pakistan  or area other then US. 
And the next thing is that temporal resolution of the SAR Data and optical dataset is different. So how the model integrate these datasets.

**Comment by lonnes** [comment, marked as answer]
Hello. SpaceEye workflow has global coverage and should work for Pakistan. Can you provide more information regarding the issues you are seeing with the generated images? 

SpaceEye generates daily cloud-free imagery based on Sentinel1 and Sentinel2. The temporal interpolation is done to harmonize between the difference time coverages of each satellite constellation. At the moment, we linearly interpolate Sentinel1 to generate a daily timeseries and then use Sentinel1 data to regenerate the Sentinel2 rasters. More information on the solution can be found in this [paper](https://arxiv.org/pdf/2106.08408.pdf).

**Comment by mursaleen-bkk** [comment]
Hello, 
Sir, I want to monitor the harvesting of the crop. My crop is sugarcane with the time period from the start of september 2022 to the feb 2023.the harvesting time of sugarcane in my dataset is november 2022,decebmer 2022, january 2023, feb 2023 but thses months have cloud cover and frost in the study area so that's why I shifted to spaceEye data. My spaceEye images for these months appear to be same. I am confused about the fact that why is there no change in the images for different months and dates. To be more precise, my spaceEye data show same images of the  different months I have mentioned above. However, change in the images is very important for crop harvest monitoring and it does not seem possible in my case.

**Comment by rafaspadilha** [comment]
> but thses months have cloud cover and frost in the study area so that's why I shifted to spaceEye data. My spaceEye images for these months appear to be same. I am confused about the fact that why is there no change in the images for different months and dates. 

This may happen if there is not enough information in the images for that time range to reconstruct the cloud-covered pixels. Besides that, frost and snow will likely affect the output of the reconstruction if they are present in multiple images.
